server/repositories/entities/reportDefinitionFields.ts

```diff
@@ -16,7 +16,7 @@ export default class ReportDefinitionFields {
   @ManyToOne('ReportDefinition', 'fields')
   reportDefinition: ReportDefinition
 
-  @ManyToOne('Field', 'id', { eager: true })
+  @OneToOne('Field', 'id', { eager: true })
```

Comment:
Again, Mark has this modelled as a ManyToOne in the diagram 🤔

Reply:
Yep, you're correct. I've got tangled up in the OneToMany and ManyToOne